hack/ci-common.sh

```diff
@@ -85,3 +87,94 @@ clamp_mss_to_pmtu() {
     iptables -t mangle -A POSTROUTING -p tcp --tcp-flags SYN,RST SYN -j TCPMSS --clamp-mss-to-pmtu
   fi
 }
+
+# copy_kubeconfig_from_kubeconfig_env_var copies the kubeconfig to apporiate location based on kind setup
+copy_kubeconfig_from_kubeconfig_env_var(){
+  case "$SHOOT_FAILURE_TOLERANCE_TYPE" in
+    node )
+      cp $KUBECONFIG example/provider-local/seed-kind-ha-single-zone/base/kubeconfig
+      cp $KUBECONFIG example/gardener-local/kind/ha-single-zone/kubeconfig
+      ;;
+    zone )
+      cp $KUBECONFIG example/provider-local/seed-kind-ha-multi-zone/base/kubeconfig
+      cp $KUBECONFIG example/gardener-local/kind/ha-multi-zone/kubeconfig
+      ;;
+    *)
+      cp $KUBECONFIG example/provider-local/seed-kind/base/kubeconfig
+      cp $KUBECONFIG example/gardener-local/kind/local/kubeconfig
+      ;;
+  esac
+}
+
+gardener_up(){
+  case "$SHOOT_FAILURE_TOLERANCE_TYPE" in
+    node )
+      make gardener-ha-single-zone-up
+      ;;
+    zone )
+      make gardener-ha-multi-zone-up
+      ;;
+    *)
+      make gardener-up
+      ;;
+  esac
+}
```

Comment:
Let's drop these functions and rather call the right targets in the respective script files. This is hard to find and kind of double maintenance.